src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -387,16 +357,14 @@ private FedoraResource getBaseVersion(final FedoraResource resource) {
             resource.getChildren().forEach(child -> {
                 if (child.getTypes().contains(WEBAC_AUTHORIZATION)) {
                     final Map<String, List<String>> aclTriples = new HashMap<>();
-                    child.getTriples(translator, PROPERTIES)
-                        .filter(p -> isAclPredicate.test(model.asStatement(p).getPredicate()))
-                        .forEach(t -> {
-                            aclTriples.putIfAbsent(t.getPredicate().getURI(), new ArrayList<>());
-                            if (t.getObject().isURI()) {
-                                aclTriples.get(t.getPredicate().getURI()).add(
-                                    substringBeforeLast(t.getObject().getURI(), "/" + JCR_CONTENT));
-                            } else if (t.getObject().isLiteral()) {
-                                aclTriples.get(t.getPredicate().getURI()).add(
-                                    t.getObject().getLiteralValue().toString());
+                    child.getTriples(translator, PROPERTIES).filter(hasAclPredicate)
+                        .forEach(triple -> {
+                            aclTriples.putIfAbsent(triple.getPredicate().getURI(), new ArrayList<>());
```

Comment:
I would take `triple.getPredicate().getURI()` out into a variable here, for readability.